base_geoengine/static/src/js/widgets/geoengine_widgets.js

```diff
@@ -207,6 +207,9 @@ odoo.define('base_geoengine.geoengine_widgets', function (require) {
             // Default extent
             if (map_view) {
                 var extent = this.defaultExtent.split(', ');
```

Comment:
could we take the opportunity to be more resilient on this split?
```suggestion
                var extent = this.defaultExtent.replace(/\s/g, '').split(',');
```